servo-media/src/backends/gstreamer/audio_sink.rs

```diff
@@ -124,7 +124,7 @@ impl AudioSink for GStreamerAudioSink {
             // sometimes nothing reaches the output
             if chunk.len() == 0 {
                 chunk.blocks.push(Default::default());
-                chunk.blocks[0].mix(channels as u8);
+                chunk.blocks[0].repeat(channels as u8);
```

Comment:
You probably want to also signal the actual channel layout to GStreamer via the channel-mask or the positions array in the GstAudioInfo

Reply:
We only support one layout in gstreamer right now, yeah?

Reply:
Oh, I see. I'll be working on this when I add better support for output channel counts. Currently it's hardcoded to be two.